### PR TITLE
Add average cycle time

### PR DESCRIPTION
**Pull request by kaipot**
From https://skyline.ms/announcements/home/support/thread.view?rowId=49861

### CODE REVIEW

File: pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs [hunk 1]
```diff
@@ -85,6 +85,17 @@ public void ChangeChromInfo(EditDescription editDescription, Func<TransitionChro
         [Format(NullValue = TextUtil.EXCEL_NA)]
         public int? PointsAcrossPeak { get { return ChromInfo.PointsAcrossPeak; } }
 
+        [Format(Formats.RETENTION_TIME, NullValue = TextUtil.EXCEL_NA)]
+        public double? AverageCycleTime
+        {
+            get
+            {
+                return StartTime.HasValue && EndTime.HasValue && PointsAcrossPeak.HasValue
```
Comment (nickshulman) [inline pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs:93]: You can simplify this by doing:
                return (EndTime - StartTime) * 60 / PointsAcrossPeak;

**Review by nickshulman — COMMENTED** [review]
I wonder if it should be called "CycleTimeAcrossPeak", so that it's clear that you're only looking at the cycle time across the peak, and not the average cycle time across the entire chromatogram.

File: pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs (hunk 1 above)
Comment (kaipot) [reply pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs]: That's useful, I didn't know that works with nullables.

**Comment by kaipot** [comment]
> I wonder if it should be called "CycleTimeAcrossPeak", so that it's clear that you're only looking at the cycle time across the peak, and not the average cycle time across the entire chromatogram.

That's a better name, I've updated it.

**Review by brendanx67 — APPROVED** [review]
Looks good.

**Pull request merged**